designer/test/new-configuration.test.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
```

Comment:
I'm not going to block the review on it but it's a bit of a shame that more of the new files you've created aren't unit tested. Is there any chance of adding coverage either in this PR or a later one?

Reply:
which files do you mean? the react ones? I was just moving code out of App and they weren't previously UT'd, and since they were from the hack day I didn't bother..! I'll open an issue for adding tests. 

As for the services, I can add tests. I'll do it in a separate PR before this one goes out of sync again. thanks!